packages/hooks/src/__tests__/useP2PConfig.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { APIProvider, useFetch } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useP2PConfig from '../useP2PConfig';
+
+type TWrapper = {
+    children: JSX.Element;
+};
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(),
+}));
+
+const mockUseFetch = useFetch as jest.MockedFunction<typeof useFetch<'website_status'>>;
+const mock_store = mockStore({
+    client: {
+        currency: 'USD',
+    },
+});
+const mock_data = {
+    data: {
+        website_status: {
+            p2p_config: {
+                payment_methods_enabled: 1,
+                adverts_active_limit: 3,
+                adverts_archive_period: 3,
+                supported_currencies: ['usd'],
+            },
+        },
+    },
+};
+
+const wrapper = ({ children }: TWrapper) => (
+    <APIProvider>
+        <StoreProvider store={mock_store}>{children}</StoreProvider>
+    </APIProvider>
+);
```

Comment:
@niloofar-deriv Let's avoid making global things and make each test case isolated, Please follow the same pattern as other test files 🙏🏻